documentation/source/users/rmg/input.rst

```diff
@@ -378,17 +416,75 @@ For example ::
 		toleranceMoveToCore=0.1,
 		toleranceInterruptSimulation=0.1,
 		maxNumObjsPerIter=2,
+		terminateAtMaxObjects=True,
 	)
 
 Note that this can also result in larger models, however, sometimes these larger models (from taking more than one
-object at a time) pick up chemistry that would otherwise have been missed.  
+object at a time) pick up chemistry that would otherwise have been missed.
 
 .. _ontheflyquantumcalculations:
 
+Advanced Setting:  Dynamics Criterion
+-------------------------------------
+While the flux criterion works very well for identifying new species that have high flux
+and therefore will likely be high throughput or high concentration species
+it provides few automatic guarantees about how well a given model will accurately represent
+the concentrations of the involved species.  The dynamics criterion is more complex
+than the flux criterion, but in general it is a measure of how much impact a given
+reaction will have on the concentrations of core species.  A more detailed explanation
+is available in the theory section.
+
+Reasonable values for this criterion range typically between 5-30.
+
+For example ::
+
+	model(
+		toleranceMoveToCore=0.1,
+		toleranceInterruptSimulation=0.1,
+		toleranceMoveEdgeReactionToCore=10.0,
+		toleranceMoveEdgeReactionToCoreInterrupt=5.0,
+	)
+
+Note that it is highly recommended to use the dynamics criterion only alongside the flux
+criterion and not by itself.
+
+Advanced Setting:  Surface Algorithm
+-------------------------------------
+One common issue with the dynamics criterion is that as can be seen in our theory
+section, it treats all core species equally.  Because of this, if the dynamics
+criterion is set too low it enters a feedback loop where it adds species and then
+since it can't get those species' concentrations right it adds more species and so on.
+In order to avoid this feedback loop the surface algorithm was developed.  It creates
+a new partition called the surface that is considered part of the core.  We will
+refer to the part of the core that is not part of the surface as the bulk core.  Operating
+without the dynamics criterion everything moves from edge to the bulk core as usual
+however the dynamics criterion is managed differently.  Lower dynamics criterion
+tolerances are set for movement from edge to surface than from edge to bulk core, with
```

Comment:
This is hard to understand I think. I might be changing the meaning of your sentence, but consider the following re-wording:

`Lower dynamics criterion tolerances are set so that species move from the edge to the surface rather than from the edge to the bulk core (with the constraint that edge reactions cannot enter the surface if they don't have either all reactants or all products in the bulk core.)`